indexers/keyvalue/MongoDBIndexer/mongodbhandler.py

```diff
@@ -83,3 +88,31 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self.client.close()
         except pymongo.errors.PyMongoError as exp:
             raise MongoDBException(exp)
+
+    def delete(self, keys: Iterator[int], *args, **kwargs):
+        import pymongo
+        try:
+            count = self.collection.delete_many({'_id': {'$in': keys}}).deleted_count
+            self.logger.debug(f'deleted {count} documents in the database')
+            return
```

Comment:
same here, I am not sure how it would play if we ever add a `finally`

Reply:
Done